packer/ovf2ova.rb

```diff
@@ -27,8 +27,7 @@
 doc.at_xpath("/xmlns:Envelope/ovf:VirtualSystem/ovf:VirtualHardwareSection/ovf:Item[rasd:ResourceType=10]/rasd:Connection").content = "NAT Network"
 doc.at_xpath("/xmlns:Envelope/ovf:VirtualSystem/ovf:VirtualHardwareSection/ovf:Item[rasd:ResourceType=10]/rasd:ResourceSubType").content = "vmxnet3"
 
-# Remove unnecessary sound card and disk controllers
-doc.at_xpath("/xmlns:Envelope/ovf:VirtualSystem/ovf:VirtualHardwareSection/ovf:Item[rasd:ResourceType=35]").remove
+# Remove unnecessary disk controllers
 doc.xpath("/xmlns:Envelope/ovf:VirtualSystem/ovf:VirtualHardwareSection/ovf:Item[rasd:ResourceType=5]").each do |node|
```

Comment:
@juju2112 was this tested with Virtualbox _and_ some VMWare variant like VMWare Player?

Reply:
@mariussturm Not until you said that. Thanks for keeping me honest!

Originally, when I tried to build this with Packer 1.6.0, it failed on line 31 above. When I opened graylog.ovf in vi, I could see in the XML file that the node the ruby script was looking for was not there. That's why it failed, because it can't remove something that doesn't exist. 

I have now spun up the OVA built from this pipeline in Virtualbox and VMWare Player, and I was able to get Graylog started and login to it successfully in both cases.

Reply:
Awesome :+1: